src/PositionManager.sol

```diff
@@ -110,23 +110,32 @@ contract PositionManager is
     using SafeTransferLib for *;
     using CurrencyLibrary for Currency;
     using PoolIdLibrary for PoolKey;
-    using PositionConfigLibrary for PositionConfig;
     using StateLibrary for IPoolManager;
     using TransientStateLibrary for IPoolManager;
     using SafeCast for uint256;
     using SafeCast for int256;
     using CalldataDecoder for bytes;
     using SlippageCheckLibrary for BalanceDelta;
-    using PositionConfigIdLibrary for PositionConfigId;
+    using PoolKeyChecker for PoolKey;
 
     /// @dev The ID of the next token that will be minted. Skips 0
     uint256 public nextTokenId = 1;
 
-    mapping(uint256 tokenId => PositionConfigId configId) internal positionConfigs;
+    // TODO: Move to custom type, packed memory.
+    struct PositionInfo {
+        // lower 25 bytes of the poolId
+        bytes25 poolId;
+        int24 tickLower;
+        int24 tickUpper;
+        bool hasSubscriber;
+    }
+
+    mapping(uint256 tokenId => PositionInfo info) internal positionInfo;
```

Comment:
public?

Reply:
unless im missing something? how will position descriptor call to fetch this information?